schedule/yast/autoyast_salt_formulas.yaml

```diff
@@ -0,0 +1,15 @@
+---
+name: autoyast_salt_formulas
+description: |
+  Test installation using AY plus salt formulas.
+  Test formulas used is a testing one from https://w3.suse.de/~igonzalezsosa/salt/
```

Comment:
not a good idea to reference his personal directory, we would just store it in data/.

Reply:
so we should probably not mention it in the description of our test.